ga4gh/backends.py

```diff
@@ -186,19 +186,19 @@ def searchVariants(self, request):
         else:
             readCols = self._table.columns()[:self._firstSamplePosition]
             callSetIds = request.callSetIds
-            for csid in request.callSetIds:
-                cols = [c for name, c in self._sampleCols[csid]]
+            for callSetId in request.callSetIds:
+                cols = [col for name, col in self._sampleCols[callSetId]]
                 readCols.extend(cols)
         # Now we get the row positions for the sample columns
```

Comment:
Nit: the nested for loops are obscure enough we should consider commenting *why* we're doing this, or perhaps how, or looking for a better control structure.